Jenkinsfile_CNP

```diff
@@ -90,8 +92,8 @@ node {
                         onDemo {
                                 withSubscription('nonprod') {
 
-                                        KeyVault keyVault = new KeyVault(this, 'nonprod', "ccd-${environment.demoName}")
-                                        initialiseLogstashConfFiles(keyVault)
+                                        KeyVault keyVault = new KeyVault(this, 'nonprod', "${productName}-${environment.nonProdName}")
```

Comment:
how come we changed environment.demoName to environment.nonProdName?

Reply:
demo is aslo nonprod